src/components/RightPane/RightPaneStore.ts

```diff
@@ -70,6 +75,28 @@ class RightPaneStore extends EventEmitter {
     toggleOpenSpotAlert = () => {
         this.openSpotAlertPopoverActive = !this.openSpotAlertPopoverActive;
     };
+
+    focusOnBuilding = (buildingFocusInfo: BuildingFocusInfo) => {
+        this.emit('focusOnBuilding', buildingFocusInfo);
+        // NOTE: This might not be accurate. Some things were lost in the rewrite.
```

Comment:
Can you double check this and delete this line about uncertainty if it's all accurate? Also, can you reference the PR number in the explanation, like "this description is accurate as of #488" or something?